src/components/UserBookmarksPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import TopBarProgress from 'react-topbar-progress-indicator';
+import PropTypes from 'prop-types';
+import UserArticleCardComponent from './common/UserArticleCardComponent';
+import {
+  getAllUsersBookMarkedArticles
+} from '../redux/actions/articleActions/articleActions';
+
+
+class UserBookmarksPage extends Component {
+  state = {
+    currentPage: 1,
+    bookmarksPerPage: 4,
+  }
+
+  handleClick= (event) => {
+    this.setState({
+      currentPage: Number(event.target.id),
+    });
+  }
+
+  componentDidMount() {
+    const { auth: { isVerified } } = this.props;
+    if (isVerified) this.props.getAllUsersBookMarkedArticles();
+  }
+
+  render() {
+    let bookmarkView;
+
+    const { currentPage, bookmarksPerPage } = this.state;
+    const { bookmarkedArticles: { isLoading, userBookmarks } } = this.props;
+
+    if (!isLoading && userBookmarks.length === 0) {
```

Comment:
Let's take advantage of javascript's falsy values
```suggestion
    if (!isLoading && !userBookmarks.length) {
```